src/core/InterleavedBufferAttribute.js

```diff
@@ -17,7 +17,7 @@ THREE.InterleavedBufferAttribute.prototype = {
 
 	get length() {
```

Comment:
```count``` works for me need to change in ```BufferAttribute``` too.

Take this change then add ```count``` and alias ```length``` to ```count``` with a deprecated warning?